src/main/java/com/linecorp/armeria/server/docs/Specification.java

```diff
@@ -50,7 +50,7 @@ static Specification forServiceEntries(List<ServiceEntry> services) {
 
             final ThriftService thriftService = thriftServiceOptional.get();
             final Optional<String> debugPath =
-                    thriftService.serializationFormat() == SerializationFormat.THRIFT_TEXT ?
+                    thriftService.allowedSerializationFormats().contains(SerializationFormat.THRIFT_TEXT) ?
```

Comment:
Yeah I realized that it'd actually be useful for DocService to display the path regardless of this debugging ability. It's a bit complicated and tangential from allowing the codec to sniff content-type, so seems ok for a followup PR just for DocService?